overlord/ifacestate/helpers.go

```diff
@@ -719,3 +726,41 @@ func ensureSystemSnapIsPresent(st *state.State) error {
 	_, err := snapstate.CoreInfo(st)
 	return err
 }
+
+func hotplugTaskSetAttrs(task *state.Task, deviceKey, ifaceName string) {
```

Comment:
These can be setHotplugAttrs and getHotplugAttrs, consistent with the names already in place for the other functions below (getHotplugSlots, and setHotplugSlots).